industrial_taxonomy/pipeline/glass_description_ngrams/nlp_utils.py

```diff
@@ -0,0 +1,182 @@
+from itertools import product
```

Comment:
Because these helper functions are coupled to the flow (i.e. they live in `pipeline/...`) and are not general utilities or getters (live in `utils/...` or `getters/...`) then we don't **need** to because no-one other than the flow should be importing these functions.
(Having said this it may be useful to add descriptions depending on the context)